Packs/Cryptosim/Integrations/Cryptosim/Cryptosim.py

```diff
@@ -0,0 +1,266 @@
+from datetime import datetime, timedelta
+from CommonServerPython import *
+from CommonServerUserPython import *
+import traceback
+import json
+import base64
+import requests
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+
+class Client(BaseClient):
+    def correlation_alerts(self, last_fetch_time=None):
+        args = demisto.args()
+
+        end_time = datetime.utcnow() + timedelta(hours=int(demisto.params().get("time_zone",3)))
+        interval_time = end_time - timedelta(minutes=int(demisto.params().get('incidentFetchInterval', 360)))
+
+        formatted_start_time = datetime.strptime(last_fetch_time, DATE_FORMAT) + timedelta(hours=int(demisto.params().get("time_zone",3))) if last_fetch_time is not None else None
+
+        if last_fetch_time is None or formatted_start_time < interval_time:
+            formatted_start_time = interval_time
+
+        if formatted_start_time >= end_time:
+            formatted_start_time = formatted_start_time - timedelta(minutes=int(demisto.params().get('incidentFetchInterval', 360)))
+
+        parameters = {
+            'startDate': args.get('startDate', formatted_start_time.isoformat()),
+            'endDate': args.get('endDate', end_time.isoformat()),
+            'showSolved': args.get('showSolved', False),
+            'crrPluginId': args.get('crrPluginId', -1),
+            'containStr': args.get('containStr', None),
+            'risk': args.get('risk', -1),
+            'srcIPPort': args.get('srcIPPort', None),
+            'destIPPort': args.get('destIPPort', None),
+            'srcPort': args.get('srcPort', None),
+            'destPort': args.get('destPort', None),
+            'riskOperatorID': args.get('riskOperatorID', "equal"),
+            "limit": int(args.get("limit",100)),
+            "isJsonLog": True
+        }
+
+        return self._http_request("POST", url_suffix="correlationalertswithlogs",
+                                  data=json.dumps(parameters))
+
+    def correlations(self):
+        args = demisto.args()
+
+        limit = str(args.get("limit",100))
```

Comment:
limit is str, 
so you need to use `arg_to_number` for converting it to int, or to set 100 to string default

Reply:
so,
limit = args.get("limit","100")
is okey for this case ?

Reply:
`str(args.get("limit", '100'))`

Reply:
I fixed it.